package.json

```diff
@@ -144,12 +145,12 @@
     "sass-loader": "7.0.3",
     "shelljs": "0.8.2",
     "style-loader": "0.21.0",
-    "ts-loader": "4.3.1",
+    "ts-loader": "4.4.2",
```

Comment:
Can your remove ts-loader from package.json (and use awesome-typescript-loader in the tests too)

Reply:
it actually causes the test run to hang and never start...